src/style.css

```diff
@@ -1,3 +1,81 @@
-body {
+:root {
+  --light-blue: #00acdc;
+}
+
+* {
   margin: 0;
+  font-family: Roboto, serif;
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+}
```

Comment:
Don't use '*' selector as it impacts performance. Set styles only for elements that require them. If you need to reset margins and paddings, consider using a more specific reset CSS or applying them to body and html tags.